doc/authors.md

```diff
@@ -31,42 +32,42 @@ Marius Kjærstad (6)
 Luke Dashjr (6)
 David Mercer (6)
 Daniel Cousens (6)
+Charlie O'Keefe (6)
 Suhas Daftuar (5)
-Pavel Janík (5)
+Peter Todd (5)
 Karl-Johan Alm (5)
 Johnathan Corgan (5)
-Charlie O'Keefe (5)
 Alex Morcos (5)
 WO (4)
 Philip Kaufmann (4)
-Peter Todd (4)
+Pavel Janík (4)
```

Comment:
-Pavel Janík (5)
-Pavel Janik (2)
+Pavel Janík (4)

I am not sure what to make of these numbers.

Reply:
I was confused by that too. I will investigate it before the next release.